### DISCUSSION TITLE
Timeout config issue

### DISCUSSION THREAD

**Discussion by alexandremjacques** [unpoly/unpoly#560] [Q&A] [answered]
Hi,

I have configured the `up.network.config.timeout = undefined`. I have a page that, sometimes , takes a very long time to load (1-3 minutes).

Even having configured the timeout, I still see a timeout error being issued in the console:

<img width="702" alt="image" src="https://github.com/unpoly/unpoly/assets/1953298/439bb10e-4f30-4b9f-8956-1c5d68787ad2">

I'm working with the 3.0.0 version. Is there some known bug that affects this config?

Is there something I can do to avoid that?

TIA

**Comment by triskweline** [comment, marked as answer]
You configuration looks good.

You can check if your configuration is being picked up:

```js
up.on('up:request:load', (event) => console.log("Timeout is", event.request.timeout))
```

**Reply by alexandremjacques** [reply]
Aparently it was not loaded beacuse of the browser cache (it was loading a previous version of the script file).

Thanks!
